cloud-sql/mysql/Startup.cs

```diff
@@ -55,152 +54,157 @@ public void ConfigureServices(IServiceCollection services)
                     options.Version = "Test";
                 });
             }
-            services.AddScoped(typeof(DbConnection), (IServiceProvider) =>
-                InitializeDatabase());
+            services.AddScoped(typeof(MySqlConnectionStringBuilder),
+                (IServiceProvider) => new MySqlConnect().GetMySqlConnectionString());
+            services.AddScoped<MySqlConnect>();
             services.AddMvc(options =>
             {
                 options.Filters.Add(typeof(DbExceptionFilterAttribute));
             });
             _services = services;
         }
 
-        DbConnection InitializeDatabase()
+        public class MySqlConnect

```

Comment:
I think I would get this class out of here, and make most of the methods extension methods on `MySqlConnectionStringBuilder` or `MySqlConnection`.

Reply:
Done.